### ISSUE TITLE
train human detection model

### ISSUE THREAD

**Issue by Oliverwang** [luoyetx/Joint-Face-Detection-and-Alignment#7]
Can this framework be used to train human detection models?   net‘s’ weight height rate isn't  1:1

**Comment by luoyetx** [comment]
Yes. The cascade framework can be used to detection objects with fixed window size.
